enterprise/cluster/src/main/java/org/neo4j/cluster/protocol/atomicbroadcast/multipaxos/InMemoryAcceptorInstanceStore.java

```diff
@@ -29,18 +31,21 @@
         implements AcceptorInstanceStore
 {
     private final Map<InstanceId, AcceptorInstance> instances;
+    private final BlockingQueue<InstanceId> currentInstances;
```

Comment:
Can you please add description what is general idea of this class and why we need instances, currentInstances fields and why they can be out of sync - as example after clear method call